impexp-core/src/main/java/org/citydb/core/operation/exporter/concurrent/DBExportWorker.java

```diff
@@ -184,6 +190,11 @@ private void doWork(DBSplittingResult work) {
 					AbstractGML object = exporter.exportObject(work.getId(), work.getObjectType());
 					if (object instanceof AbstractFeature) {
 						feature = (AbstractFeature) object;
+
+						if (useLodFilter && feature.isSetBoundedBy()) {
+							feature.setBoundedBy(feature.calcBoundedBy(bboxOptions));
```

Comment:
The feature here is top-level. How about sub-features, should their bbox also be recalculated?

Reply:
e8097757143c409f6f67d3a2da7650d71fd52644